### ISSUE TITLE
[FEATURE] default event for actionHelper should be configurable

### ISSUE THREAD

**Issue by runspired** [emberjs/ember.js#10090]
LinkView's eventName can be changed via a reopen, the action handler cannot be easily changed, this makes it more difficult than necessary to create an addon that switches out Ember's default eventing.  In my case, I've overridden the default of "click" with "tap" on linkViews.  I would like to do the same with the action helper without needing to entirely overwrite the helper.

**Comment by trek** [comment]
Can you tell us what you'd like to customize?

**Comment by runspired** [comment]
This request really has two parts.

I wrote (ember-mobile)[https://github.com/runspired/ember-mobiletouch] Touch to be a dropin addon for enabling full gesture support with an ember-centric approach to eventing.  This includes removing "click" as the default action and linkView event in favor of "tap".

**part 1: make it easy to change the event that triggers an action from "click" to "tap".**

For the action helper, this requires overwriting the helper like below (See here: https://github.com/runspired/ember-mobiletouch/blob/master/app/initializers/ember-mobiletouch.js#L22-L27).

```javascript
var oldActionHelper = Ember.Handlebars.helpers.action;
    Ember.Handlebars.helpers.action = function (params, hash, options, env) {
      var opts = options ? options : hash.hash;
      opts.on = opts.on || 'tap';
      return oldActionHelper.apply(null, arguments);
    }
```

**part 2: make it possible to listen to more than one event with an action helper**

Often on mobile more than one event should trigger the same behavior.  For instance, iOS style slide-toggles trigger on "tap" "press" or "pan".

The most critical instance of this is to allow more than one gesture to initiate a tap, because many users (especially older ones) conflate "tapping" with "pressing".  So as a default, I'm working towards allowing either a "tap" or a "press" event to trigger "linkView" or an action.

**Comment by runspired** [comment]
Incidentally, I'd be open to PRing these changes if acceptable, they shouldn't require very much.

**Comment by rwjblue** [comment]
I think that this can be done via a AST plugin. Basically it would walk the templates, and actually rewrite it as if 'on' had been specified. This would work with 1.10 and up....

**Comment by thatikuntap** [comment]
is this issue fixed in 1.13

**Comment by rwjblue** [comment]
FYI - ember-mobiletouch was updated to use an AST transform to provide a default action if none was specified.
